src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
+     */
+    @Setter
+    @Getter
+    @ToString
+    @Data
```

Comment:
:warning: Using @Data annotation from Lombok can lead to generating excessive methods, violating SRP. (Severity: Major)